portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase

```diff
@@ -0,0 +1,70 @@
+@component-name = "portal-smoke"
+definition {
+
+	property testray.main.component.name = "Smoke";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			PagesAdmin.tearDownCP();
+
+			Site.tearDownCP();
+
+			WebContent.tearDownCP();
+		}
+	}
+
+	@description = "Add tests for LPS-148855, ensure guest group roles are inherited by the Guest user."
```

Comment:
This is a test for LPS-148855. The default site role shouldn't be inherited by Guest.